src/Maestro/Maestro.Data/Models/Build.cs

```diff
@@ -106,6 +107,7 @@ public class BuildChannel
         public Build Build { get; set; }
         public int ChannelId { get; set; }
         public Channel Channel { get; set; }
+        public DateTimeOffset DateAdded { get; set; }
```

Comment:
Why not use a nullable here?  All of the DateAdded items will end up invalid for any old builds, but not using a nullable will make it difficult to disambiguate.

Reply:
They will all be DateTimeOffset.MinValue, that is not much different than null, and non-nullable here makes the code easier.

Reply:
Okay fine by me then.

Reply:
Done

Reply:
> Done

I agree with @alexperovich on the reasons to leave it as non-nullable (potentially cleaner code where used since there is no null check), so you can revert your changes if you wish

Reply:
I missed all these comments. Undone.